README.md

```diff
@@ -76,22 +75,18 @@ $socket = new SecureServer($socket, $loop, array(
 $http = new React\Http\Server($socket);
 ```
 
-For each incoming connection, it emits a `request` event with the respective
+For each incoming connection, it executes the callback function with the respective
 [`Request`](#request) and [`Response`](#response) objects:
```

Comment:
@legionth The code above does not seem to take care of this, can you update this? Also make sure the README is in line with the respective classes :+1: